include/TrustWalletCore/TWTezosAddress.h

```diff
@@ -0,0 +1,50 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "TWBase.h"
+#include "TWData.h"
+#include "TWString.h"
+
+TW_EXTERN_C_BEGIN
+
+struct TWPublicKey;
+
+static const size_t TWTezosAddressSize = 27;
```

Comment:
Should this be 22 as specified for the size of bytes?

Reply:
3 prefix + 20 blakeb + 4 check